mechanics/tools/gdocs-migrate/main.go

```diff
@@ -3,13 +3,15 @@ package main
 import (
 	"fmt"
 	"log"
+	"strings"
 
 	"google.golang.org/api/drive/v3"
 )
 
 const (
-	KnativeCommunityDriveId = "0APnJ_hRs30R2Uk9PVA"
-	KnativeTeamDriveId      = "0AM-QGZJ-HUA8Uk9PVA"
+	KnativeCommunityDriveID = "0APnJ_hRs30R2Uk9PVA"
+	KnativeTeamDriveID      = "0AM-QGZJ-HUA8Uk9PVA"
+	FolderMimeType          = "application/vnd.google-apps.folder"
```

Comment:
I may add a flag that would be a doc ID prefix to allow us to test this without hitting the whole folder. WDYT?

Reply:
That sounds good to me. But this runs really fast and so far I've not seen 429 🤘